config.py

```diff
@@ -16,6 +16,8 @@ class BaseConfig:
     GTM_CONTAINER_ID = os.getenv('GTM_CONTAINER_ID')
     GTM_TAG_ID = os.getenv('GTM_TAG_ID')
 
+    FORCE_HTTPS = True
```

Comment:
I'm not sure about this, I think the nginx is handling ssl termination and calling the http endpoint of the pod?